src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -26,6 +26,43 @@
 
 namespace Microsoft.ML.Transforms
 {
+    /// <summary>
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="NormalizingTransformer"/>.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | <xref:System.Single> or<xref:System.Double> or a known-sized vector of those types. |
```

Comment:
```suggestion
    /// | Input column data type | <xref:System.Single> or <xref:System.Double> or a known-sized vector of those types. |
```

Reply:
thank you for catching the space.

---
In reply to: [277110149](https://github.com/dotnet/machinelearning/pull/3432#discussion_r277110149) [](ancestors = 277110149)